### DISCUSSION TITLE
WSL integration not working

### DISCUSSION THREAD

**Discussion by larryclaman** [rancher-sandbox/rancher-desktop#2037] [Q&A] [answered]
I'm using Rancher Desktop on Windows 11, and on the Windows side it's working fine.  However, I cannot seem to get the WSL integration to work.
Looking in the config screen, it's not letting me select either of my WSL instances.  Any suggestions for how to fix this or where I should be looking?

![rancher](https://user-images.githubusercontent.com/25161683/163453825-71362ec0-bc53-4ca4-8d85-e70f694872f7.png)

**Comment by mook-as** [comment, marked as answer]
Given the description by the check boxes, I assume you need to look at the kubeconfig files in the individual WSL distros and determine if you want to keep them.  Rancher Desktop wants to replace them with symlinks (so they stay up to date when you restart Rancher Desktop), but deleting your existing configuration files automatically wouldn't be great.

Please check if you're happy with deleting them manually, and do that if it seems fine.

Something like:
```
PS> wsl -d Ubuntu
$ ls -la ~/.kube/config
$ cat ~/.kube/config
$ rm ~/.kube/config # if you're okay with deleting the file
```

**Reply by larryclaman** [reply]
that was it! Thanks!

**Reply by vedrancu** [reply]
How will kubectl work if I delete config file?

**Reply by bhanuprakash-Csg** [reply]
kubectl is cli  but kubeconfig is authenticate the cluster
